CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.0.0 (Apr 20, 2021)
```

Comment:
This should be 0.17.0. Maybe we need to revisit the date?

Reply:
of course! my bad